shears/project-updates-noel.yml

```diff
@@ -0,0 +1,48 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
Could you change this comment to summarize the purpose of this? I'm not too familiar with GitHub Actions or Docker so it's not really clear to me.